lib/cluster.rb

```diff
@@ -120,6 +120,14 @@ def brt_counts
     @brt_counts
   end
 
+  def ch_counts
```

Comment:
I'm not sure what status "ch" means -- consider changing the name of the method to reflect that? Likewise, consider changing `wd_counts` below to `withdrawn_counts`, and consider extracting a `counts_by_status` method perhaps?

Reply:
I think there are enough little reasons for not doing this _right now under the scope of HT-1944_.
* An empty status counts towards `ch_counts`, but not to `lm_counts` or `wd_counts`.
* `brt_counts` kind of looks the same but is counting something else.
* I could get rid of e.g. "wd" in `wd_counts` by making `withdrawn_counts` and having it pass `"wd"` as an argument to `counts_by_status`. I don't see the improvement. Is it just that the method names would make more sense?
* There are no tests for `(brt|lm|wd)_counts`.

Reply:
That's fine -- yes, the main benefit is just that the method names would make more sense and that it would reduce duplication. Empty status counting towards `ch_counts` but not `lm_counts` or `wd_counts` is probably enough reason not to worry about extracting a method right now.